src/containers/ConceptPage/components/ConceptMetaData.jsx

```diff
@@ -7,19 +7,30 @@
  */
 
 import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
 import { injectT } from '@ndla/i18n';
 import FormikField from '../../../components/FormikField';
 import {
   FormikLicense,
   FormikContributors,
   FormikMetaImageSearch,
 } from '../../FormikForm';
-import { LicensesArrayOf } from '../../../shapes';
+import { LicensesArrayOf, SubjectShape } from '../../../shapes';
+import { MultiSelectDropdown } from '../../../components/Dropdown';
 
 const contributorTypes = ['creators'];
 
-const ConceptMetaData = ({ t, licenses }) => (
+const ConceptMetaData = ({ t, licenses, subjects, tags }) => (
   <Fragment>
+    <FormikField
+      name="tags"
+      label={t('form.tags.label')}
+      description={t('form.tags.description')}
+      obligatory>
```

Comment:
Ikkje obligatorisk. Kan også fjerne description i og med at det heller ikkje er krav om tre nøkkelord for concept.